src/Microsoft.DotNet.Interactive.Jupyter/KernelExtensions.cs

```diff
@@ -22,10 +22,10 @@ namespace Microsoft.DotNet.Interactive.Jupyter
 {
     public static class KernelExtensions
     {
-        public static Task UseJupyterHelpersAsync<TKernel>(this TKernel kernel) where TKernel : DotNetKernel
+        public static Task UseJupyterHelpersAsync<TKernel>(this TKernel kernel) where TKernel : ISupportSetValue
```

Comment:
Maybe pull in #1489 here? This really should not be async.

Reply:
that Pr is still open and not working.